src/pages/studyView/UserSelections.tsx

```diff
@@ -318,6 +323,54 @@ export default class UserSelections extends React.Component<
             components
         );
 
+        // Generic Assay chart filters
+        _.reduce(
+            this.props.filter.genericAssayDataFilters || [],
+            (acc, genericAssayDataFilter) => {
+                const uniqueKey = getGenericAssayChartUniqueKey(
+                    genericAssayDataFilter.stableId,
+                    genericAssayDataFilter.profileType
+                );
+                const chartMeta = this.props.attributesMetaSet[uniqueKey];
+                if (chartMeta) {
+                    acc.push(
```

Comment:
lets assign to a variable and then push it at the end to avoid nesting.  tough to read

Reply:
Not sure if this looks better?